R/Dockstore.R

```diff
@@ -36,9 +36,11 @@ Dockstore <-
     }
     .Dockstore(
         Service(
-            "dockstore", host="dockstore.org", authenticate = FALSE,
-            api_url = "https://dockstore.org/swagger.json",
-            api_reference_md5sum = "1ea8b7659454180eb83598d6bd2b4d01"
```

Comment:
When present the md5sum (on the yaml) has been used to validate that the version we have an interface to is the version that dockstore is using. Removing the md5sum removes this validation? It looks like there's a version field in the yaml. Is it updated regularly by the dockstore folks? can we use that instead?

Reply:
AFAIU, the md5sum was used for API json/yaml files that were used as-is. The previous Dockstore API was using Swagger 2.0 where the md5sum would match the file downloaded from the `api_url`. Now they are using OpenAPI 3.0 so the `Service` function call below matches the Leonardo, Terra, and Rawls APIs. We are converting OpenAPI 3.0 to Swagger 2.0 and the md5sums are not expected to match across this conversion.